web/client/epics/identify.js

```diff
@@ -7,13 +7,26 @@
 */
 const Rx = require('rxjs');
 
-const {LOAD_FEATURE_INFO, GET_VECTOR_INFO} = require('../actions/mapInfo');
+const {LOAD_FEATURE_INFO, GET_VECTOR_INFO, FEATURE_INFO_CLICK} = require('../actions/mapInfo');
 const {closeFeatureGrid} = require('../actions/featuregrid');
+const {CHANGE_MOUSE_POINTER, CLICK_ON_MAP} = require('../actions/map');
+const {MAP_CONFIG_LOADED} = require('../actions/config');
+const {stopGetFeatureInfoSelector} = require('../selectors/mapinfo');
 
 module.exports = {
     closeFeatureGridFromIdentifyEpic: (action$) =>
         action$.ofType(LOAD_FEATURE_INFO, GET_VECTOR_INFO)
         .switchMap(() => {
             return Rx.Observable.of(closeFeatureGrid());
+        }),
+    changeMapPointer: (action$, store) =>
```

Comment:
not sure about this epic, can you explain it?

Reply:
It could throw the change-map-pointer action before the map is configured so the mouse pointer action will fail

Reply:
maybe add a comment above this epic with the answer you give me. :)